src/ex1/index.html

```diff
@@ -1,14 +1,22 @@
+
```

Comment:
Product review:
First of all good job!
1. You didn't do all the requirements of the exercise, please go over it again
2. The ctrl+z is an **amazing** addition, however, if I wouldn't look at your code I wouldn't have any idea that this functionality is available for me, try to think how you can expose it to the user
3. You could go a little bit further with the design, it is part of the exercise
4. I wonder if it would also be nice to have a button that clears all the tasks as well